### DISCUSSION TITLE
can't use cli firebase mac pro m1

### DISCUSSION THREAD

**Discussion by AhkamKhallaf** [firebase/flutterfire#8633] [Q&A]
i'm trying to add firebase using cli , when i run  flutterfire configure
 ,it says zsh: command not found: flutterfire
, i use mac pro m1 as operating system , 
flutter doctor doesn't give me any errors ?

**Comment by crllnsmnn** [comment]
What helped me was to run `export PATH="$PATH":"$HOME/.pub-cache/bin"` and then `dart pub global activate flutterfire_cli` and finally `flutterfire configure` worked.

**Reply by holeg** [reply]
Thanks, that was the solution for me too

**Reply by EB301** [reply]
was  for me also 👍🏾

**Reply by GarifullinII** [reply]
Thank you! It's good!

**Reply by jineshsoni** [reply]
Perfect, thanks

**Reply by AmazinglyStrange** [reply]
Weeping, weeping... I've tried several things but this simple direction made the magic happen. Thank you!

**Reply by shadhin96** [reply]
thanks

**Comment by abdalmonem** [comment]
for mac m1 , add export PATH="$PATH":"$HOME/.pub-cache/bin" at the top of users/username/.profile , works on (monterey)
